s3tests_boto3/functional/test_sts.py

```diff
@@ -200,8 +198,6 @@ def test_get_session_token_permanent_creds_denied():
     assert response['ResponseMetadata']['HTTPStatusCode'] == 200
     
     s3_client=boto3.client('s3',
-                aws_access_key_id = s3_main_access_key,
-		aws_secret_access_key = s3_main_secret_key,
```

Comment:
hmm, this causes the `create_bucket()` call to fail with
> botocore.exceptions.NoCredentialsError: Unable to locate credentials

Reply:
@cbodley : As part of STSLite (predecessor to STS) in rgw, `getsessiontoken` was implemented to reduce the load on Keystone/LDAP or any external server. The implementation was AWS non-compliant and the idea was to authenticate `GetSessionToken` with the external server using the EC2/LDAP creds and return temporary creds, and these temp creds could be used for further s3 ops.  This idea was extended to Local Auth also, hence this test case covers a failure scenario for local auth. I do not think anybody is using/used STS Lite, so we can either ignore this failure, mark this as fails_on_rgw or just remove it.